src/display.asm

```diff
@@ -86,3 +86,17 @@ print_string:
 .done:
   pop ax
   ret
+
+render_controlls:
+  push dx
+  mov dx, 0x0000
+  mov si, left_string
```

Comment:
The current calling convention is to push and pop all registers that are manipulated by the callee. This should therefore also happen to the `si` register.